app/mailers/invite_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class InviteMailer < Devise::Mailer
+
+  protected
+
+  def subject_for(key)
+    if key.to_s == 'invitation_instructions'
+      "New Sanctuary Coalition (#{ @resource.community.name.upcase }) volunteer account invitation"
+    else
+      return super
+    end
```

Comment:
Happy to refactor the above block if there's a different style you'd like to go for here, was going off the docs for simplicity.